google-cloud-storage/src/test/java/com/google/cloud/storage/BlobTest.java

```diff
@@ -662,4 +666,50 @@ public Long answer() throws Throwable {
     byte actual[] = Files.readAllBytes(file.toPath());
     assertArrayEquals(expected, actual);
   }
+
+  private WriteChannel createWriteChannelMock(byte[] bytes) throws Exception {
+    WriteChannel channel = createMock(WriteChannel.class);
+    ByteBuffer expectedByteBuffer = ByteBuffer.wrap(bytes, 0, bytes.length);
+    expect(channel.write(expectedByteBuffer)).andReturn(bytes.length);
+    channel.close();
+    replay(channel);
+    return channel;
+  }
```

Comment:
It's just my observation, I mostly see private/utility methods at the bottom of the Test class.

As I said it's just my observation, if you feel strongly about it,  no need to refactor this.

Reply:
Do you mean instead of:
```
private void helper() {...}
@Test pulic void testA() {...}
@Test pulic void testB() {...}
```
should be:
```
@Test pulic void testA() {...}
@Test pulic void testB() {...}
private void helper() {...}
```
?

Reply:
yes exactly

Reply:
All test cases are usually in sequence then helper methods should be bundled

Reply:
I haven't found any recommendations about that. My personal feeling about that:
- generic helpers (like setup, init, etc) should be located at the begining 
- helpers used by a group of tests should be located next that group

So, I personally prefer to keep that method where it currently is. If you don't mind :)

Reply:
That's why I wrote my observation, It works either way for me.

Reply:
Thanks for review!